src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -174,6 +174,19 @@ public void verifyIsEmailOfInstructorOfCourse(String instructorEmail, String cou
         }
     }
 
+    public void verifyAtLeastOneInstructorIsDisplayed(boolean isOriginalInstructorDisplayed, String courseId,
+                                                      boolean isEditedInstructorDisplayed)
```

Comment:
Nitpick: why is the order of variable `boolean, String, boolean`? Not wrong, but kind of weird.